notebooks/writing_a_trace_transform_cpu_offloading.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        first_used (bool): Whether to return the map of first used variable to symbol mapping if True otherwise return the map for last used.
</pre><p>ditto (indent)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Lightning-AI/lightning-thunder/pull/966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>